### PR TITLE
Added setting to exclude URLs from autourlconf

### PR DESCRIPTION
**Pull request by benbacardi**

### CODE REVIEW

File: django_autoconfig/app_settings.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 #: Extra URLs for autourlconf
 AUTOCONFIG_EXTRA_URLS = getattr(settings, 'AUTOCONFIG_EXTRA_URLS', ())
 
+#: A list/tuple of apps to exclude from the autourlconf
+AUTOCONFIG_EXCLUDE_URLS = getattr(settings, 'AUTOCONFIG_EXCLUDE_URLS', ())
```
Comment (mikebryant) [inline django_autoconfig/app_settings.py]: I'm not a fan of the name here, the one above is about adding URLs, this is about excluding apps from the urlconf
Maybe `AUTOCONFIG_URLCONF_EXCLUDE_APPS`?

File: django_autoconfig/app_settings.py [hunk 2]
```diff
@@ -5,6 +5,9 @@
 #: Extra URLs for autourlconf
 AUTOCONFIG_EXTRA_URLS = getattr(settings, 'AUTOCONFIG_EXTRA_URLS', ())
 
+#: A list/tuple of apps to exclude from the autourlconf
```
Comment (mikebryant) [inline django_autoconfig/app_settings.py]: Will this be surfaced in the docs? I don't think we're currently displaying the autogenerated page from this file

File: django_autoconfig/app_settings.py (hunk 1 above)
Comment (mikebryant) [inline django_autoconfig/app_settings.py]: Looking further, I think my conclusion is the other setting is badly named... Let's try and make this one better :p

File: django_autoconfig/app_settings.py (hunk 2 above)
Comment (benbacardi) [inline django_autoconfig/app_settings.py]: I've added a line to the docs.

**Pull request merged**